samza-api/src/main/java/org/apache/samza/system/SystemAdmin.java

```diff
@@ -28,6 +28,17 @@
  * utility methods that Samza needs in order to interact with a system.
  */
 public interface SystemAdmin {
+
+  /*
+   * Start SystemAdmin
```

Comment:
"Start/Stop this SystemAdmin"

Reply:
Sure. I added "this" to the Java doc of start() and stop().